src/Workspaces/CSharp/Portable/Classification/SyntaxClassification/CSharpSyntaxClassificationService.cs

```diff
@@ -17,6 +17,7 @@ internal class CSharpSyntaxClassificationService : AbstractSyntaxClassificationS
     {
         private readonly ImmutableArray<ISyntaxClassifier> s_defaultSyntaxClassifiers =
             ImmutableArray.Create<ISyntaxClassifier>(
+                new EmbeddedLanguageTokenClassifier(),
```

Comment:
📝 String escape sequences are not an embedded language. The entire use of the embedded language subsystem makes classification of a core syntactical element of C# overly complex.